src/main/docs/guide/modules-testcontainers.adoc

```diff
@@ -147,3 +147,33 @@ test-resources:
 
 The `network` key is used to declare the network containers use.
 The `network-aliases` can be used to assign host names to the containers.
+
+== Wait strategies
+
+Micronaut Test Resources uses the default https://www.testcontainers.org/features/startup_and_waits/[Testcontainers wait strategies].
+
+It is however possible to override the default by configuration.
+
+[source,yaml,subs="verbatim"]
+----
+test-resources:
+  containers:
+    mycontainer:
+        image-name: my/container
+        wait-strategy:
+            log:
+                regex: ".*Started.*"
+----
+
+The following wait strategies are supported:
+
+- `log` strategy with the following properties:
+    - `regex`: the regular expression to match
+    - `times`: the number of times the regex should be matched
+- `http` strategy with the following properties:
+    - `path`: the path to check
+    - `tls`: if TLS should be enabled
+    - `port`: the port to listen on
+    - `status-code`: the list of expected status codes
+- `port` strategy (this is the default strategy)
```

Comment:
Do you mean it’s a default strategy for the custom containers? Most of the JDBC containers have SQL teat query except for Oracle and Postgres which come with the log check.

Reply:
It's the default strategy that is applied by test containers on generic containers.